### ISSUE TITLE
Calling this.import(...) results in a TypeError

### ISSUE THREAD

**Issue by buschtoens** [ember-engines/ember-engines#544]
I could not find any explicit docs regarding importing vendor files / npm packages in ember-engines, but extrapolating from the [ember-cli docs](https://ember-cli.com/extending/#importing-dependency-files) and the following warning, I assume that calling `this.import(...)` in the `included()` hook is the correct way.

https://github.com/ember-engines/ember-engines/blob/16f20c958afde91d7e24f8786a6e74a05467cf6d/lib/engine-addon.js#L578-L580

```js
const EngineAddon = require('ember-engines/lib/engine-addon');

module.exports = EngineAddon.extend({
  name: 'heilmittelkatalog',

  lazyLoading: true,

  included() {
    this._super.included && this._super.included.apply(this, arguments);
    this.import('node_modules/cycled/index.js', {
      using: [{ transformation: 'cjs', as: 'cycled' }]
    });
  },

  isDevelopingAddon() {
    return true;
  }
});
```

However, I get this TypeError:

```
TypeError: Cannot read property 'set' of undefined
    at Class.import (/home/jan/gitlab/web/frontend/app/node_modules/ember-cli/lib/broccoli/ember-app.js:1659:25)
    at Class.import (/home/jan/gitlab/web/frontend/app/node_modules/ember-engines/lib/engine-addon.js:562:28)
    at Class.included (/home/jan/gitlab/web/frontend/app/lib/heilmittelkatalog/index.js:22:16)
    at Class.superWrapper [as included] (/home/jan/gitlab/web/frontend/app/node_modules/ember-cli/node_modules/core-object/lib/assign-properties.js:34:20)
    at Class.included (/home/jan/gitlab/web/frontend/app/node_modules/ember-engines/lib/engine-addon.js:604:28)
    at project.addons.project.addons.filter.addon (/home/jan/gitlab/web/frontend/app/node_modules/ember-cli/lib/broccoli/ember-app.js:538:17)
    at Array.filter (<anonymous>)
    at EmberApp._notifyAddonIncluded (/home/jan/gitlab/web/frontend/app/node_modules/ember-cli/lib/broccoli/ember-app.js:535:47)
    at new EmberApp (/home/jan/gitlab/web/frontend/app/node_modules/ember-cli/lib/broccoli/ember-app.js:164:10)
    at module.exports (/home/jan/gitlab/web/frontend/app/ember-cli-build.js:10:15)
```

https://github.com/ember-cli/ember-cli/blob/v3.0.2/lib/broccoli/ember-app.js#L1659

Excerpt of the host project's `package.json`:

```js
{
  "devDependencies": {
    "cycled": "^0.2.0",
    "ember-cli": "~3.0.0",
    "ember-cli-babel": "^6.12.0",
    "ember-cli-cjs-transform": "^1.1.1",
    "ember-engines": "^0.5.14",
    "ember-source": "^2.18.2",
    // ...
  },
  "ember-addon": {
    "paths": [
      "lib/heilmittelkatalog"
    ]
  }
}
```

As you can see, this is an `in-repo-engine`, in case this is relevant.

**Comment by aaxelb** [comment]
FYI, this is fixed for ember-cli 3.1 by #562, but I don't know if it'll be merged because it doesn't work with ember-cli master.

**Comment by villander** [comment]
it's fixed by #630
